3rdparty/libjpeg-turbo/CMakeLists.txt

```diff
@@ -149,7 +149,7 @@ endif()
 if(MSVC)
   option(WITH_CRT_DLL
     "Link all ${CMAKE_PROJECT_NAME} libraries and executables with the C run-time DLL (msvcr*.dll) instead of the static C run-time library (libcmt*.lib.)  The default is to use the C run-time DLL only with the libraries and executables that need it."
-    FALSE)
+    TRUE)
```

Comment:
`TRUE` is not a valid value too:
- we have static builds too - e.g., `BUILD_SHARED_LIBS=OFF`
- we have dedicated flag to control that - `BUILD_WITH_STATIC_CRT`

We should drop this option from 3rdparty completely as we manage common flags on OpenCV level - see OpenCVCRTLinkage.cmake

The similar note is about changing of other common flags.

Reply:
Resolved.